### ISSUE TITLE
vita2d_font_draw_text() emoji's

### ISSUE THREAD

**Issue by devingDev** [xerpi/vita2dlib#50]
I have tried several fonts with the vita2d_font. I took ".ttf"'s which support emoji , and tested all. It seems it can handle stuff like this ❤️  which was this unicode : "\u2764"
But when trying something else like 🚐 (": minibus:") which is this unicode(s) : "\ud83d\ude90"  it does not work. They show up as squares or something like that or something else.
So emojis with only one unicode sequence work whereas the emojis consisting of more show as some symbols. I tested the ttf and they do display those just not on the psvita.

Here some more examples: 

"\ud83c\udf54" => 🍔 
"ud83d\udda5" => 🖥
"ud83c\udf1a" => 🌚 
"\u2764" => ❤️ 

Some relevant sites: 
http://www.utf8-chartable.de/unicode-utf8-table.pl?start=128640&view=2
http://unicode.org/emoji/charts/full-emoji-list.html
https://github.com/MorbZ/OpenSansEmoji

**Also** Maybe I should instead parse the strings for unicode and replace them with emoji pngs , what do you say?

Edit: http://unicodelookup.com/#🍔/1   => u+1f354

**Comment by devnoname120** [comment]
Emojis shouldn't be included by default or it will bloat vita2d.

For example, the [EmojiOne](https://www.emojione.com/developers/download) [32x32px pack](https://d1j8pt39hxlh3d.cloudfront.net/emoji/emojione/3.0/EmojiOne_3.0_32x32_png.zip) takes 2.34 MB.

**Comment by devingDev** [comment]
You are right and I was not actually wanting a actual png support. I mean the fonts when I use them on Windows display emojis. Emojis are just some utf32 code(s) and some fonts display them.
They are not as fancy as emoji pngs but well at least something. Anyway I solved this issue by making an emoji spritesheet and generating a json for that. Now I just parse the strings via utf32 converter and display emojis where they belong although it is not trivial to position them as glyph size is not equal always between characters.


For me this case is closed. I just wanted to bring up the fact that fonts do display emojis, although not as good as image.
